test/test.js

```diff
@@ -92,7 +91,6 @@ describe('Test coloredcoinsd', function () {
     }
     cc.broadcastTx(args, function (err, ans) {
       assert(err)
```

Comment:
This is not returning an error any more? because this is a transaction that already on the block-chain